vendor.conf

```diff
@@ -1,5 +1,9 @@
 github.com/agl/ed25519 d2b94fd789ea21d12fac1a4443dd3a3f79cda72c
 github.com/Azure/go-ansiterm 19f72df4d05d31cbe1c56bfc8045c96babff6c7e
+github.com/bugsnag/bugsnag-go 13fd6b8acda029830ef9904df6b63be0a83369d0
+github.com/bugsnag/osext 0dd3f918b21bec95ace9dc86c7e70266cfc5c702
+github.com/bugsnag/panicwrap e2c28503fcd0675329da73bf48b33404db873782
+github.com/BurntSushi/toml a368813c5e648fee92e5f6c30e3944ff9d5e8895
```

Comment:
This is bringing in a whole lot of unnecessary dependencies.

I believe the problem is `github.com/docker/notary/util`. I think this illustrates one of the major problems with a `utils` package. 

from https://blog.golang.org/package-names

> Avoid meaningless package names. Packages named util ...

A generic package like `utils` tends to collect a bunch of unrelated functions/types that have completely different dependencies which means that using any of the functions results in the consumer having to depend on a bunch of unrelated things (sql libraryes, logging libraries, etc).

I think this needs to be fixed in notary and anything used in `docker/cli` needs to be moved out of `utils` so we can use it without bringing in a dozen unnecessary packages.

Reply:
I'm looking into it - it's possible that this may not be so clean because the notary server-side services also use a key storage interface with connects with the DB (hence many of the imports).  I'll get back to you with an update.